### PR TITLE
DND Module release.

### PR DESCRIPTION
**Pull request by ghost**
Module based on Anti PM and AFK modules.

### CODE REVIEW

File: dnd.py [hunk 1]
```diff
@@ -0,0 +1,454 @@
+# -*- coding: future_fstrings -*-
+
+#    Friendly Telegram (telegram userbot)
+#    By Magical Unicorn (based on official Anti PM & AFK Friendly Telegram modules)
+#    Copyright (C) 2020 Magical Unicorn
```
Comment (penn5) [inline dnd.py:5]: This isn't right, because it's not copyright you.

File: dnd.py [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+from telethon import functions, types
+logger = logging.getLogger(__name__)
+
+
+def register(cb):
+    cb(DoNotDisturb())
+
+
+@loader.tds
+class DoNotDisturb(loader.Module):
```
Comment (penn5) [inline dnd.py:35]: DoNotDisturbMod

File: dnd.py [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+                          "<a href='https://github.com/LegendaryUnicorn/FTG-Unofficial-Modules'>Github</a>."),
+               "who_to_allow": "<b>Who shall I allow to PM ?</b>",
+               "who_to_block": "<b>Specify who to block.</b>",
+               "who_to_deny": "<b>Who shall I deny to PM ?</b>",
+               "who_to_report": "<b>Who shall I report ?</b>",
+               "who_to_unblock": "<b>Specify who to unblock.</b>"}
+
+    def __init__(self):
+        self._me = None
+        self.default_pm_limit = 50
```
Comment (penn5) [inline dnd.py:95]: That's a configuration value surely

File: dnd.py [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+
+    def config_complete(self):
+        self.name = self.strings["name"]
+
+    async def client_ready(self, client, db):
+        self._db = db
+        self._client = client
+        self._me = await client.get_me(True)
+
+    async def afkbackcmd(self, message):
```
Comment (penn5) [inline dnd.py:105]: Old namings were better - unafk

File: dnd.py [hunk 5]
```diff
@@ -0,0 +1,454 @@
...
+        """Remove the AFK status.\n """
+        self._db.set(__name__, "afk", False)
+        self._db.set(__name__, "afk_gone", None)
+        self._db.set(__name__, "afk_rate", [])
+        await utils.answer(message, self.strings["afk_back"])
+
+    async def afkgocmd(self, message):
```
Comment (penn5) [inline dnd.py:112]: Old naming - afkcmd

File: dnd.py [hunk 6]
```diff
@@ -0,0 +1,454 @@
...
+        """
+        .afkgo : Enable AFK status.
+        .afkgo [message] : Enable AFK status and add a reason.
+         
+        """
+        if utils.get_args_raw(message):
```
Comment (penn5) [inline dnd.py:118]: Avoid double-evaluation.

**Review by penn5 — CHANGES_REQUESTED** [review]

**Comment by AvinashReddy3108** [comment]
What do we do now?
The guy deleted his GitHub account.

**Comment by HitaloM** [comment]
@AvinashReddy3108 LOL

**Pull request closed without merging**